src/Uno.UI.Maps/themes/Generic.xaml

```diff
@@ -0,0 +1,39 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+					xmlns:converters="using:Uno.UI.Converters"
+					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+					xmlns:maps="using:Windows.UI.Xaml.Controls.Maps" 
+					xmlns:mapsPresenter="using:Windows.UI.Xaml.Controls.Maps.Presenter" 
+					mc:Ignorable="d">
+
+    <!-- Default style for Windows.UI.Xaml.Controls.Maps.MapControl -->
+    <Style TargetType="maps:MapControl">
```

Comment:
So... the default MapControl Style is defined in a different NuGet package? I have a feeling this will no longer work once Styles handling is aligned with UWP.

Reply:
Why would it ? There can be a default style for a control in another package, as long as there is only one, and the map app's precedence will take over.